### PR TITLE
polyfill for statusbar

### PR DESCRIPTION
**Pull request by yawetse**
adding a dummy statusbar to increase native component compatibility

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-web",
-  "version": "0.3.2",
+  "name": "@yawetse/react-web",
```
Comment (yuanyan) [inline package.json]: Should not PR this change.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "0.3.21",
```
Comment (yuanyan) [inline package.json]: Should not PR this change too.

Comment (yawetse) [inline package.json]: fixed those, sorry about that.

**Comment by yawetse** [comment]
Can you publish a new version of react-web on NPM when you get a chance?

Thanks!

**Pull request merged**